<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1450</width>
    <height>880</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1450</width>
    <height>880</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1450</width>
    <height>880</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Газпромнефть.Автоматизация</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(0, 64, 119)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>105</y>
      <width>600</width>
      <height>670</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: 'white';
border: 3px solid 'white';
border-radius: 35px</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GillSans</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>ВХОД В ЛИЧНЫЙ КАБИНЕТ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="email_edit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>511</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GillSans</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(0, 112, 186);
border: 2 px solid rgb(0, 112, 186);
border-radius: 25px;
color: 'white'</string>
     </property>
     <property name="text">
      <string>Введите E-mail</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_edit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>330</y>
       <width>511</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GillSans</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(0, 112, 186);
border: 2 px solid rgb(0, 112, 186);
border-radius: 25px;
color: 'white'</string>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Введите пароль</string>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="sing_in_btn">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>480</y>
       <width>230</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GillSans</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: 'white';
background: rgb(0, 112, 186);
border: 2px solid rgb(0, 112, 186);
border-radius: 20px
}

QPushButton:pressed {
background: rgb(0, 82, 156);
border: 2px solid rgb(0, 82, 156)
}</string>
     </property>
     <property name="text">
      <string>ВОЙТИ</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>560</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GillSans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ваша организация не зарегистрирована?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="register_btn">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>570</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GillSans</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: rgb(0, 112, 186)
}

QPushButton:pressed {
color: rgb(0, 82, 156)
}</string>
     </property>
     <property name="text">
      <string>Зарегистрируйте!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sing_in_admin_btn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>610</y>
       <width>361</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GillSans</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: rgb(0, 112, 186)
}

QPushButton:pressed {
color: rgb(0, 82, 156)
}</string>
     </property>
     <property name="text">
      <string>Войти как сотрудник ООО «Газпромнефть-Хантос»</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>10</y>
       <width>501</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Image_dir/logos_reg.png</pixmap>
     </property>
    </widget>
    <widget class="QCheckBox" name="remember_me">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>410</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GillSans</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Запомнить меня</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1450</width>
      <height>880</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Image_dir/фон.jpg</pixmap>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>frame</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
